src/components/pages/bc-lnfi.ts

```diff
@@ -0,0 +1,69 @@
+import { customElement, state } from 'lit/decorators.js';
+import { BitcoinConnectElement } from '../BitcoinConnectElement';
+import { withTwind } from '../twind/withTwind';
+import { html } from 'lit';
+import '../internal/bci-button';
+import { classes } from '../css/classes';
+import store from '../../state/store';
+import { genericConnectorTitle } from '../connectors/bc-generic-nwc-connector';
+
+@customElement('bc-lnfi')
+export class LnfiNWCPage extends withTwind()(BitcoinConnectElement) {
+  @state()
+  private _nwcUrl = '';
+
+  override render() {
+    return html`<div class="w-full">
+      <bc-navbar class="flex w-full" heading="Micro node Wallet Connect"></bc-navbar>
+      <div class="font-sans text-sm w-full">
+        <div class="px-8 pt-4 w-full">
+          <div class="mb-1 ${classes['text-neutral-secondary']}">
+            Enter your
```

Comment:
Could you maybe add a link or extra info on how the user gets the connection string from micro node? see the mutiny connector for an example.

Otherwise, it looks good to me :rocket:

Reply:
I had add help link like mutiny.